vita3k/modules/SceSysmem/SceSysmem.cpp

```diff
@@ -62,6 +62,7 @@ struct SysmemState {
     std::mutex mutex;
     Blocks blocks;
     Blocks vm_blocks;
+    std::map<SceUID, char[KERNELOBJECT_MAX_NAME_LENGTH + 1]> mem_block_names;
```

Comment:
No need to add a new map for this

Reply:
In first i have added it inside BlockInfo, but after check in henckaku blockinfo no have name, but is inside other structure called  BlockInfoDetail
Ask @bookmist

Reply:
Just find a way to add it to block while keeping the MemBlockInfo structure the same. They are many different ways to do this (inheritance, creating a structure containing the block info and the name mainly)

Reply:
ok i no see i have no idea how done this

Reply:
is done